tasks/docker_py_install.yml

```diff
@@ -1,16 +1,29 @@
 ---
 - block: # Install docker-py
+  - name: Install dependency
+    package:
+      name: 
+      - python-setuptools
```

Comment:
make these variable, since it looks like it could be changed for each environment.